new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import io.kubernetes.client.openapi.ApiException;
+import oracle.weblogic.kubernetes.actions.impl.TraefikParams;
+import oracle.weblogic.kubernetes.actions.impl.primitive.HelmParams;
+import oracle.weblogic.kubernetes.annotations.tags.MustNotRunInParallel;
+import oracle.weblogic.kubernetes.annotations.tags.Slow;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import oracle.weblogic.kubernetes.extensions.Timing;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static oracle.weblogic.kubernetes.actions.TestActions.createIngress;
+import static oracle.weblogic.kubernetes.actions.TestActions.createUniqueNamespace;
+import static oracle.weblogic.kubernetes.actions.TestActions.deleteNamespace;
+import static oracle.weblogic.kubernetes.actions.TestActions.helmGetExpectedValues;
+import static oracle.weblogic.kubernetes.actions.TestActions.helmList;
+import static oracle.weblogic.kubernetes.actions.TestActions.installTraefik;
+import static oracle.weblogic.kubernetes.actions.TestActions.uninstallIngress;
+import static oracle.weblogic.kubernetes.actions.TestActions.uninstallTraefik;
+import static oracle.weblogic.kubernetes.actions.TestActions.upgradeTraefik;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.traefikIsRunning;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.awaitility.Awaitility.with;
+
+// this is an example of a test suite (class) where the tests need to be run in a certain
+// order. this is controlled with the TestMethodOrder annotation
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Simple validation of Traefik functions")
```

Comment:
Are we really just testing Traefik functions?  I'm suspecting that we are testing that you can load balance requests to WebLogic cluster instances.

Reply:
This is agreed with Vanaja. I can add the e2e tests if you want.

Reply:
I don't think I was clear enough.  I was suggesting the opposite.  That we don't need to test Traefik.  This test suggests that it is validating Traefik, rather than validating WebLogic.

Reply:
added create mii domain test to verify the load balancer can route the requests to the cluster.